palace.pyx

```diff
@@ -1158,6 +1160,97 @@ cdef class SourceGroup:
         """
         self.impl.destroy()
 
+
+cdef class AuxiliaryEffectSlot:
+    """An effect processor.
+
+    It takes the output mix of zero or more sources,
+    applies DSP for the desired effect (as configured
+    by a given Effect object), then adds to the output mix.
+
+    Parameters
+    ----------
+    context : Optional[Context]
+        The context from which the auxiliary effect slot is to be created.
+        If it is `None`, the object is left uninitialized.
```

Comment:
Seemingly `AuxiliaryEffectSlot(None)` isn't called anywhere, so just `Context` is enough.  This line can be accordingly removed.